openmls/src/group/tests/test_commit_validation.rs

```diff
@@ -266,23 +265,24 @@ fn test_valsem201(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider
         }))
     };
 
-    let psk_proposal = || {
-        let secret = Secret::random(ciphersuite, backend, None).unwrap();
-        let rand = backend
-            .rand()
-            .random_vec(ciphersuite.hash_length())
-            .unwrap();
-        let psk_id = PreSharedKeyId::new(
-            ciphersuite,
-            backend.rand(),
-            Psk::External(ExternalPsk::new(rand)),
-        )
-        .unwrap();
-        psk_id
-            .write_to_key_store(backend, ciphersuite, secret.as_slice())
-            .unwrap();
-        queued(Proposal::PreSharedKey(PreSharedKeyProposal::new(psk_id)))
-    };
+    // TODO(#1330)
```

Comment:
Can you add to the comment why this isn't working anymore and what's happening in #1330 to change this? Same below.